src/app/components/task-details/task-details.component.html

```diff
@@ -0,0 +1,14 @@
+@if (this.task) {
```

Comment:
no need to use this. inside the template

Reply:
I am using the if to display a message like this:
![image](https://github.com/user-attachments/assets/0a538653-64ce-4ee1-a913-e9c1c8bc88e0)
But I can remove the checking for passed in title as the not passing anything in redirects you to the 404